pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "snpe_reputation_systems"
+version = "0.0.1"
+description = "Dynamic Structural Modeling and Approximate Bayesian Inference of Online Reputation System"
+authors = ["Shrabastee Banerjee", "Narendra Mukherjee", "M. Amin Rahimian"]
```

Comment:
@Fernando-Iscar @DiSanchz - since this is to do with the authors of the codebase rather than the paper, you guys should add your names as well

Reply:
Names added, thanks! 
@narendramukherjee